get-started/fine-tuning/platform.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Fine-tuning with the kluster.ai platform
+description: Learn how to create custom models tailored to your specific tasks by fine-tuning foundation models with your own data using the kluster.ai visual interface.
+---
+
+# Fine-tuning with the kluster.ai platform
+
+The [kluster.ai platform](https://platform.kluster.ai/){target=\_blank} provides a visual, no-code approach to fine-tuning AI models. With an intuitive interface and real-time feedback, you can train customized models without writing a single line of code.
+
+This guide walks you through the platform's fine-tuning workflow, from uploading your training data to deploying your specialized model.
+
+## Prerequisites
+
+Before getting started with fine-tuning, ensure you have the following:
+
+--8<-- 'text/kluster-api-onboarding.md::1'
+- **Prepared dataset**: You need data formatted according to kluster.ai's requirements for fine-tuning (detailed below).
+
+## Supported models
+
+kluster.ai currently supports fine-tuning for the following models:
+
+--8<-- 'text/get-started/fine-tuning-supported-models.md'
+
+## Data preparation
+
+High-quality, well-formatted data is crucial for successful fine-tuning:
+
+- **Format**: Data must be in JSONL format, where each line is a valid JSON object representing a training example.
+- **Structure**: Each JSON object should contain a `messages` array with system, user, and assistant messages.
+- **Example format**:
+
+    ```json
+    --8<-- 'code/get-started/fine-tuning/prepare.json'
+    ```
+
+
+- **Quantity**: The minimum requirement is 10 examples, but more diverse and high-quality examples yield better results.
+- **Quality**: Ensure your data accurately represents the task you want the model to perform.
+
+!!! tip "Data preparation"
+    For a detailed walkthrough of data preparation, see the [Fine-tuning Sentiment Analysis Tutorial](https://docs.kluster.ai/tutorials/klusterai-api/finetuning-sent-analysis/#get-the-data){target=_blank}.
+
+!!! example "Find Llama datasets on Hugging Face"
+    There is a wide range of datasets suitable for Llama model fine-tuning on [Hugging Face Datasets](https://huggingface.co/datasets?sort=trending&search=llama){target=_blank}. Browse trending and community-curated datasets to accelerate your data preparation.
+
+## Fine-tuning workflow
+
+The process of fine-tuning a model using the kluster.ai platform interface involves several key steps:
```

Comment:
Is this on purpose or there is a bullet list here missing?

Reply:
I think it's rather referring to the next section headers etc, I don't think we need a bullet point list here.